pages/about.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/styles.css" rel="stylesheet">
+  <link href="../styles/about.css" rel="stylesheet">
+  <title>JFD Ada Personal Portfolio about</title>
 </head>
-<body>
-  
+<body class="container">
+  <header>
+    <h1>Joanna Dudley</h1>
+    <h4 class="header-subtitle">about me</h4>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="index.html">Homepage</a></li> 
+      <li><a href="#">About Me</a></li> 
+      <li><a href="portfolio.html">Portfolio</a></li> 
+    </ul>
+  </nav>
+  <main class="nested-container">
+    <div id="teapot-pic"><img src="../images/teacup-pic.png" alt="a picture of a white and blue patterned teacup"></div>
+    <div id="about-me">
+      <p>Cat ipsum dolor sit amet, unwrap toilet paper. Being gorgeous with belly side up head nudges . I rule on my back you rub my tummy i bite you hard meow and walk away. Go into a room to decide you didn't want to be in there anyway scratch leg; meow for can opener to feed me but kitty time but purr for no reason hack up furballs, as is my right stretch out on bed. Meow meow you are my person so here is a dead bird lick paws clean then retire to the warmest spot on the couch to claw at the fabric before taking a catnap please stop looking at your phone and pet me get scared by doggo. Leave fur on people's clothes you call this cat food but this cat happen now, it was too purr-fect!!! </p>
+      <p>Destroy house in 5 seconds. Please let me outside pouty face yay! wait, it's cold out please let me inside pouty face oh, thank you rub against mommy's leg oh it looks so nice out, please let me outside again the neighbor cat was mean to me please let me back inside. Eat person's food ignore the squirrels, you'll never catch them anyway and side-eyes your "jerk" other hand while being petted. Meow until belly rubs, hide behind curtain when vacuum cleaner is on, roll over, too cute for human to get mad for you have cat to be kitten me right meow or chew slippers. Flex claws on the human's belly and purr like a lawnmower rub face on everything. Rub against person because nose is wet.</p>
+    </div>
```

Comment:
When I put everything through the html checker, it threw up errors for any section elements that didn't have a header, and told me that section elements were required to have some kind of <h#> tag in them. I had a bunch of sections where it didn't make sense for there to be a header (images, links, etc) so I changed them all to div so I could pass the html checker. Is that not best practice?

Reply:
Some like the image I'm wondering if we can style without wrapping in the div, others we'd have to take a look at together, but if it's a section with text, it might be worth considering giving it a heading to make the site/content structure more discoverable for screen readers.